src/runtime-rs/crates/resource/src/share_fs/mod.rs

```diff
@@ -38,13 +38,29 @@ pub const PASSTHROUGH_FS_DIR: &str = "passthrough";
 const RAFS_DIR: &str = "rafs";
 
 #[async_trait]
-pub trait ShareFs: Send + Sync {
+pub trait ShareFs: Send + Sync + Debug {
```

Comment:
Why is `Debug` required?

Reply:
Adding the `Debug` is for debugging purposes when developing this feature. And it is not required for now. I will remove it from the code.